src/components/menus/BottomMenu.module.scss

```diff
@@ -8,6 +8,28 @@
   font-family: $font-secondary;
   letter-spacing: 0.1em;
   width: 100%;
+  position: relative;
+  overflow: hidden;
+
+  > * {
+    position: relative;
+    z-index: 2;
+  }
+
+  &::before {
+    content: '';
+    display: block;
+    position: absolute;
+    right: 6%;
+    bottom: -10px;
+    z-index: 1;
+    width: 560px;
+    height: 390px;
+    background-color: #EDECEB;
```

Comment:
should we add the color to the variables files since it will be used in more than just footer?

Reply:
Nope very good point, I think I left it out because I thought this was the only place but you're right, it's used in other places for the palm!